Tests/AmpersandTests/FontProviderTests.swift

```diff
@@ -172,4 +172,25 @@ extension FontProviderTests {
         XCTAssertEqual(black.fontName, "Avenir-Black")
         XCTAssertEqual(black.pointSize, 18)
     }
+
+    func testItVendsTheExpectedFontsForGivenStyleAndWeights() {
+        given("The font provider is configured with a custom font", closure: givenTheFontProviderIsConfiguredWithACustomFont)
+
+        let bodyRegular = fontProvider.font(forTextStyle: .body, weight: .regular, compatibleWith: .defaultContentSizeTraitCollection)
+        let bodyLight = fontProvider.font(forTextStyle: .body, weight: .light, compatibleWith: .defaultContentSizeTraitCollection)
+        let bodyMedium = fontProvider.font(forTextStyle: .body, weight: .medium, compatibleWith: .defaultContentSizeTraitCollection)
+        let bodyHeavy = fontProvider.font(forTextStyle: .body, weight: .heavy, compatibleWith: .defaultContentSizeTraitCollection)
+        let bodyBlack = fontProvider.font(forTextStyle: .body, weight: .black, compatibleWith: .defaultContentSizeTraitCollection)
```

Comment:
I think it's worth varying the content size categories here to make sure that the vended font scales accordingly.

Reply:
https://github.com/darjeelingsteve/Ampersand/pull/1/commits/3be15f61bef3b9afad7b498812152c700421110e